emark/utils.py

```diff
@@ -116,6 +118,71 @@ def handle_data(self, data: str) -> None:
         if self.root:
             self.root.children.append(data)
 
+    @classmethod
+    def from_html(cls, html: str) -> HTML2TextParser:
+        """Parse HTML and return a HTML2TextParser instance.
+        >>> HTML2TextParser.from_html(html).text
+        """
+        parser = cls()
+        parser.feed(html)
+        return parser
+
+    @classmethod
+    def to_text(cls, html: str) -> str:
+        """Parse HTML and return plain text. Similar to Gmail.
+        >>> HTML2TextParser.to_text(html)
+        """
+        return str(cls.from_html(html))
```

Comment:
I would drop this method, it's too similar to `__str__`. I think it could cause more confusion, then add benefit. Don't you agree?

Reply:
yeap, we can just use str(HTML2TextParser) to accomplish same thing